src/payment/create-payment-strategy-registry.ts

```diff
@@ -166,6 +166,17 @@ export default function createPaymentStrategyRegistry(
         )
     );
 
+    registry.register(PaymentStrategyType.AMAZONPAYV2, () =>
```

Comment:
@vmparra i don't understand this change, is this part of another ticket?

Reply:
It is not, I just sort the registry entries